src/ResourceManager/Network/Commands.Network/ChangeLog.md

```diff
@@ -20,6 +20,7 @@
 
 ## Current Release
 * Updated cmdlet New-AzureRmVirtualNetworkGatewayConnection with support for switch ExpressRouteGatewayBypass
+* Added new HostedWorkloads property for PSNetworkInterface
```

Comment:
@wdehrich  It does not seem that the altest changes in Azure:network-september-release have been ,erged here. Can you merge the upstream branch and update the PR?